pyitt/LICENSE

```diff
@@ -1,28 +0,0 @@
-BSD 3-Clause License
-
-Copyright (c) 2023, Egor Suldin
-
-Redistribution and use in source and binary forms, with or without
-modification, are permitted provided that the following conditions are met:
-
-1. Redistributions of source code must retain the above copyright notice, this
-   list of conditions and the following disclaimer.
```

Comment:
The licenses that are already used for ittapi do not cover the pyitt code. Therefore, I'm not fully sure that the license for pyitt can be simply removed.

Reply:
I doubt that we should keep one more license in `ittapi` component. Moreover, I think, one component = one license in the root. But Egor is right and we must include his name/copyright about him into generalized code ownership. I don't know what the practice of doing this?

Reply:
Do we have any legal guidance/confirmation on this? 
_But Egor is right and we must include his name/copyright about him into generalized code ownership. I don't know what the practice of doing this?_

I believe due to sanctions we can't use yandex.ru email addresses in our source code